src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,36 @@
+.stopwatch {
+
+  &__minutes {
+    @include arrow(200px, #c25d5d);
+
+    animation: arrow-movement 3600s linear infinite;
+  }
+
+  &__seconds {
+    @include arrow(300px, $color-green);
```

Comment:
You have already created a variable for 300px

Reply:
Yep, thanks, forget to change it). All done now.